styles.css

```diff
@@ -14,77 +13,285 @@ nav {
 	position: fixed;
 	top: 20px;
 }
-
 .centerNavBar {
 	background-color: #1c1c1c;
 	border-radius: 30px;
-	width: 80%;
+	width: 1000px;
 	margin: 0 auto;
 	overflow: auto;
 	padding: 10px;
-	text-align:center;
 }
-
-		
-nav li {
+nav li{
+	list-style:none;
+}		
+a {
 	font-family: 'Lobster', 'cursive';
 	 float:left;
 	 color: #ffffff;
-     list-style: none;
+	 text-decoration: none;
 	 margin-right: 90px;
 	 font-size: 1.5rem;
- }
- 
- nav ul {
+} 
+nav ul {
 	 overflow:auto;
  	 margin:0 auto;
 	 width:675px;
-  }
- 
-nav li:hover {
+}
+a:hover {
 	color:#b69a46;
-	} 
+} 
 header {
 	width: 100%;
-	height:730px;
+	height:600px;
  	background-size:cover;
  	background-image: url("http://getwallpapers.com/wallpaper/full/b/8/8/654243.jpg");
 }
-header h1{'
+header h1{
 	font-family: 'Allan'," cursive";
 	padding-top:270px;
 	text-align: center;
 	font-size:4rem;
-	color: #ffffff;	
-	padding-bottom:30px;	
+	color: #ffffff;		
 }
 .myName{
 	color:#b69a46;
 }
-
 section {
 	overflow:auto;
-	height:730px;
 	width:80%;
 	margin: 0 auto;
 }
-
-.h1About{
+.titleAbout{
 	font-family: 'Lobster', 'cursive';
```

Comment:
dont need to set font again as you have already set this font for 'a' tags

Reply:
what you mean? to choose another font?